tox.ini

```
@@ -15,32 +15,30 @@ addopts = --cov help_tokens --cov-report term-missing --cov-report xml
 norecursedirs = .* requirements
 
 [testenv]
-deps =
-	django22: Django>=2.2,<2.3
-	django30: Django>=3.0,<3.1
-	django31: Django>=3.1,<3.2
-	django32: Django>=3.2,<3.3
-	-r{toxinidir}/requirements/test.txt
+deps = 
+    django32: Django>=3.2,<4.0
+    django40: Django>=4.0,<4.1
+    -r{toxinidir}/requirements/test.txt
 commands = 
-	python -Wd -m pytest {posargs}
+    python -Wd -m pytest {posargs}
 
 [testenv:quality]
-setenv =
-	DJANGO_SETTINGS_MODULE = test_settings
+setenv = 
+    DJANGO_SETTINGS_MODULE = test_settings
 whitelist_externals = 
-	make
-	rm
-	touch
+    make
+    rm
+    touch
 deps = 
-	-r{toxinidir}/requirements/quality.txt
-	-r{toxinidir}/requirements/test.txt
+    -r{toxinidir}/requirements/quality.txt
+    -r{toxinidir}/requirements/test.txt
 commands = 
-	touch tests/__init__.py
-	pylint help_tokens tests
-	rm tests/__init__.py
-	pycodestyle help_tokens tests
-	pydocstyle help_tokens tests
-	python setup.py check --restructuredtext --strict
-	isort --check-only --recursive tests help_tokens manage.py setup.py test_settings.py
-	make selfcheck
+    touch tests/__init__.py
+    pylint help_tokens tests
+    rm tests/__init__.py
+    pycodestyle help_tokens tests
+    pydocstyle help_tokens tests
+    python setup.py check --restructuredtext --strict
+    isort --check-only --recursive tests help_tokens manage.py setup.py test_settings.py
```

`recursive` flag isn't needed anymore with latest version.
```suggestion
    isort --check-only tests help_tokens manage.py setup.py test_settings.py
```